pkgs/by-name/ho/hobbes/package.nix

```
@@ -14,34 +13,20 @@
 }:
 llvmPackages.stdenv.mkDerivation {
   pname = "hobbes";
-  version = "0-unstable-2023-06-03";
+  version = "0-unstable-2025-04-23";
 
   src = fetchFromGitHub {
     owner = "morganstanley";
     repo = "hobbes";
-    rev = "88a712b85bc896a4c87e60c12321445f1cdcfd00";
-    hash = "sha256-2v0yk35/cLKTjX0Qbc8cjc7Y6bamRSa9GpPvGoxL2Cw=";
+    rev = "0829030be03d47f91075cbebd0c9565f44bf9911";
+    hash = "sha256-GZ26XL4++2MWQED2tVWeJ8HFcFQUXnXZ3+JCgdlAXNo=";
   };
 
-  patches = [
-    # fix build for LLVM-12+
-    # https://github.com/morganstanley/hobbes/pull/452
-    (fetchpatch {
-      name = "include-cstdint.patch";
-      url = "https://github.com/morganstanley/hobbes/commit/924b71fca06c61e606792cc8db8521fb499d4237.patch";
-      hash = "sha256-/VsWtTYc3LBOnm4Obgx/MOqaaWZhUc8yzmkygtNz+mY=";
-    })
+  CXXFLAGS = [
```

```suggestion
  env.CXXFLAGS = [
```

Better for arbitrary env vars.

---

That wouldn't let me use a list.

---

I _guess_.. if you want it for the neatness. It all gets squashed to a string anyway :shrug: